packages/patternfly-4/react-core/src/components/Select/Select.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/components/FormControls/styles.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /** additional classes added to the Select control */
+  className: PropTypes.string,
+  /** value of selected option */
+  value: PropTypes.any,
+  /** Array of options OR array of option Groups available to select from.  If getOptionGroupLabel, it should be an array ofoption groups */
+  options: PropTypes.array,
+  /** flag indicating the options are grouped */
+  isOptionsGrouped: PropTypes.bool,
```

Comment:
That is fine. how about 
getOptionsFromGroup -> getGroupOptions (since it returns a list)


getOptionDisabled -> getDisabled ( I think this one could be tricky.  I want it to be clear that it is the option that is disabled and not the whole input.)